### ISSUE TITLE
Expose prefixApi as an import

### ISSUE THREAD

**Issue by blade2005** [ecyrbe/zodios-express#130]
I'm working with `@zodios/openapi` and my api is set up using prefixes.

```
export const usersApiBuilder = apiBuilder({
  method: "get",
  path: "/",
  response: z.array(UserResponseSchema),
  alias: "getUsers",
  description: "Get users",
  parameters: DefaultPathParameters,
  errors: [
    {
      status: 404,
      schema: NotFoundErrorSchema
    },
    {
      status: "default",
      schema: ErrorSchema
    }
  ]
});
export const usersApi = usersApiBuilder.build();
const userRouter = ctx.router(usersApi);
app.use("/users", userRouter);
const document = openApiBuilder({
  title: "Zodios PoC Express Serverless",
  version: "1.0.0",
  description: "A simple user API"
})
  .addServer({ url: "/api/v1" })
  .addSecurityScheme("cognito", bearerAuthScheme())
  .addProtectedApi("cognito", usersApi)
  .build();
```

I would need to wrap `usersApi` as `prefixApi("/users", usersApi)` when passing it to `addProtectedApi`

I found `prefixApi` but it doesn't seem to be exported for use as an import. I guess I could recreate it in my code but that seems wasteful

**Comment by ecyrbe** [comment]
Hello,

Yes, i created this, but was not sure it was the right abstraction so i did not exposed it yet.

So i'll certainly open a discussion about how we could improve route composition on server side and api composition for definition.

minor version incoming to expose this utility for now.

**Comment by blade2005** [comment]
I appreciate that a lot. One thing I did notice in looking at it is changing the `Api` generic like this `Api extends readonly ZodiosEndpointDefinition[]`

**Comment by ecyrbe** [comment]
it's available on version `v10.5.0` of `@zodios/express`
